.editorconfig

```diff
@@ -0,0 +1,9 @@
+root = true
+
+[*]
+charset = utf-8
+indent_style = space
+indent_size = 4
```

Comment:
NAB: Just a opinion that should be open to discussion with the owner, airbnb-base uses indent_size = 2.

Reply:
My current IDE settings using 4 by default for javascript. So, that's ok for me.